pytest_docker_tools/factories/container.py

```diff
@@ -8,16 +8,49 @@
 def container(request, docker_client, wrapper_class, **kwargs):
     """ Docker container: image={image} """
 
+    wrapper_class = wrapper_class or Container
+
+    if request.config.option.reuse_containers:
+        if "name" in kwargs.keys():
+            name = kwargs["name"]
+            current_containers = docker_client.containers.list(ignore_removed=True)
+            for cont in current_containers:
+                if cont.name == name:
+                    if (
+                        "Pytest-Docker-Tools.Reusable_Container"
+                        in cont.attrs["Config"]["Labels"]
+                        and cont.attrs["Config"]["Labels"][
+                            "Pytest-Docker-Tools.Reusable_Container"
+                        ]
+                        == "True"
+                    ):
```

Comment:
I try to avoid [sadface dedent](https://lukasz.langa.pl/1d1a43c4-9c8a-4c5f-a366-7f22ce6a49fc/) where possible for if's, I just find it a bit hard to read, especially when there is already a lot of indentation. Maybe make a is_reusable_container helper so you have a bit more line length to play with? And make a constant for `Pytest-Docker-Tools.Reusable_Container` thats a bit shorter?

Reply:
done